AlphaWallet/Rpc Network/AvailableRpcNetwork.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  AvailableRpcNetwork.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 20/12/21.
+//
+
+import Foundation
+
+@available(iOS 13.0, *)
+fileprivate let SelectedCompressionAlgorithm = NSData.CompressionAlgorithm.lzma
+
+fileprivate let ChainsZipFileName = "chains_mini.zip"
+fileprivate var chains: [CustomRPC]?
+
+struct RpcNetwork: Codable {
+    let name: String
+    let chainId: Int
+    let shortName: String
+    let networkId: Int
+    let nativeCurrency: NativeCurrency
+    let rpc: [String]
+    let faucets: [String]
+    let infoURL: String
+
+    var isTestNet: Bool {
+        name.lowercased().contains("testnet")
+    }
+    var customRpc: CustomRPC {
+        CustomRPC(chainID: chainId, nativeCryptoTokenName: nativeCurrency.name, chainName: name, symbol: nativeCurrency.symbol, rpcEndpoint: rpc.first ?? "", explorerEndpoint: infoURL, etherscanCompatibleType: .unknown, isTestnet: isTestNet)
+    }
+
+    static func availableServers(fileName: String = ChainsZipFileName) -> [CustomRPC]? {
```

Comment:
For functions like this that don't cause any state changes (basically static func + not change class vars and their function arguments), can you try writing them as like described as "Functional core"  in https://github.com/AlphaWallet/alpha-wallet-ios/issues/2918#issuecomment-870350388 and the few comments after that? It's a little odd and 1 more indentation, but it helps to group them together, filtering out other code that causes state changes.